src/main/features/claimant-response/views/defendants-response.njk

```diff
@@ -1,10 +1,6 @@
 {% extends "layout.njk" %}
 
-{% if claim.response.responseType === ResponseType.PART_ADMISSION.value and page === 1 %}
-  {% set heading = 'How they want to pay' %}
-{% else %}
-  {% set heading = 'The defendant’s response' %}
-{% endif %}
+{% set heading = 'The defendant’s response' %}
```

Comment:
the code you removed, is that not necessary? we don't need different heading for PART_ADMISSION? 

i didn't find anything in the story to make this change...could you please explain??

Reply:
Ok well the story didn't explain things and was incorrect so I have left notes on the story but in general the change is to remove the additional pagination (the extra page) in the view defendant response for party type individual and company only for part admission and redirect to task list page